app/models/cis2_info.rb

```diff
@@ -44,6 +44,10 @@ def can_prescribe_pgd?
     role_code == NURSE_ROLE
   end
 
+  def can_prescribe_psd?
```

Comment:
If we're adding a new permission, we should also add it to https://github.com/nhsuk/manage-vaccinations-in-schools/blob/62ad5d3e976b274360ed4b5ea1d7a13bdc8e119e/app/controllers/concerns/authentication_concern.rb#L51 to make sure that a user who only has permission to do that and nothing else will continue to be able to log in.